index.yaml

```diff
@@ -101,6 +101,13 @@ indexes:
   - name: last_updated
     direction: desc
 
+- kind: FeedbackMessageModel
```

Comment:
Sorry, you mean -- it doesn't automatically get added back when you poke around and test etc.? If so, that's fine, possibly it's just not even needed, and it's OK to leave deleted. (On the other hand, if you find that it auto-updates when you play around with the dev server / tests, then it's OK to leave it in.)

Giving LGTM on the assumption that it doesn't get auto-added and you don't actually need to leave it in, but if you do then please feel free to regenerate it and then submit.